cfme/tests/v2v/test_v2v_cancel_migrations.py

```diff
@@ -4,97 +4,92 @@
 import fauxfactory
```

Comment:
@sshveta  can you replace ```Test to validate basic navigations.Later to be replaced with End-to-End functional testing.``` with some meaningful test title.

Reply:
:+1: